Formula/kertish-dfs.rb

```diff
@@ -0,0 +1,30 @@
+class KertishDfs < Formula
+  desc "Kertish FileSystem and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dfs"
+  url "https://github.com/freakmaxi/kertish-dfs/archive/v21.2.0066.tar.gz"
+  sha256 "1ff300a803a0d3bd068e1492319a771912cccd4ba3a72098b6d05678813af966"
+  license "GPL-3.0-only"
+  head "https://github.com/freakmaxi/kertish-dfs.git"
+
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "mod", "download"
```

Comment:
What does this download?

Reply:
When I do not call go mod download before building the source, it is throwing an exception related to missing packages. That's why I have to call it to set up the environment to build

Reply:
According to the documentation, you are allowed to use cargo, pip, and the like to download the libraries to build the source code. `go mod download` has a similar approach. Even you use `go.sum` you will need to call `go mod download` to make the libraries available to build. I'm not using `go.sum` because `go.sum` is freezing the version of the package and creating recurring bugs and force me to push a new release of the source continuously even there is no change in my code. 

What I see that you want me to put the `vendor` folder in the repository, which contains all the libraries to build the source, that is the most common mistake that has been done by go engineers because they do not want to take the responsibility for the discontinued packages.

I believe that, if you allow using cargo or pip to build rust or python code on the fly, you would think to allow `go mod download` for the same approach to build go code.

What is your thought about it? I do not want to challenge you in here but try to create awareness of the usage of `go mod download` command.

Reply:
Anything we install using `pip` are declared in `resource` blocks in the formula (see, for example, `ansible`). Similarly, we pass the [`--locked` flag to `cargo build`](https://rubydoc.brew.sh/Formula.html#std_cargo_args-instance_method) so that we use the versioned dependencies declared in the `Cargo.lock` file. (And encounter errors when there is no `Cargo.lock`.)

It sounds like your dependencies here aren't fully versioned. If so, then that would be unacceptable in homebrew/core. You can host this formula in your own tap, of course.

Reply:
Okay, if versioning is so important in homebrew, I'll create a new release with `go.sum`...

Reply:
Yes, every other Go formula in homebrew-core using Go modules includes a `go.sum`. This follows [Go's own recommendations](https://blog.golang.org/using-go-modules). Most package managers require reproducible builds so if there wasn't a `go.sum` then we would require `resource` blocks for every dependency. In fact, Go themselves sees this important enough to have made a change in Go 1.16 that _requires_ `go.sum` unless you explicitly tell it not to (e.g. `go mod download`).

The new release looks good, so you should be able to remove this line now. :+1: